.github/scripts/checkMembership.mjs

```diff
@@ -10,34 +10,31 @@ const octokit = new Octokit({
     }
 });
 
-async function main() {
-    let prAuthor;
-
+async function isMemberOfOrganization(username) {
+    let isMember = false;
     try {
-        const context = github.context;
-        const owner = context.repo.owner;
-        const repo = context.repo.repo;
-        prAuthor = context.payload.pull_request.user.login;
-
-        // Check if the PR author is a member of the organization
-        const isMember = await octokit.orgs.checkMembershipForUser({
-            org: owner,
-            username: prAuthor,
+        const members = await octokit.paginate(octokit.orgs.listMembers, {
```

Comment:
Not urgent, but should this do a loop just in case we grow in the future?

Reply:
This should return all members according to the docs, it just returns them 100 entries at a time (the default is 30)